### PR TITLE
odps-func: add page

### PR DESCRIPTION
**Pull request by onesuper**
odps func command(like the concept of subcommand) deals with a set of commands dedicated to function management.

This page serves as supplementary to odpscmd(https://github.com/tldr-pages/tldr/blob/master/pages/common/odps.md)

Other subcommands like odps-instance, odps-auth... will come in other pages.

### Note

e.g. all odps commands can be executed within console OR externally, like:

```
odpscmd -e 'list resources;'
```

### CODE REVIEW

File: pages/common/odps-func.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- Show functions in the current project:
+
+`list functions;`
+
+- Create a Java function using a jar resouce:
+
+`create function {{func_name}} as {{path.to.package.Func}} using '{{package.jar}}';`
+
+- Create a Python function using a py resource:
```
Comment (waldyrious) [inline pages/common/odps-func.md]: "py resource" sounds awkward to me. Would ".py file" work? Or we could say ".py resource" (and ".jar resource" above, to match).

Comment (onesuper) [inline pages/common/odps-func.md]: sounds reasonable

**Comment by waldyrious** [comment]
I don't think commit 2cb618c was an improvement. I'd suggest reverting that one.

**Comment by onesuper** [comment]
Revert done.

File: pages/common/odps-func.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+`list functions;`
+
+- Create a Java function using a .jar resource:
+
+`create function {{func_name}} as {{path.to.package.Func}} using '{{package.jar}}';`
+
+- Create a Python function using a .py resource:
+
+`create function {{func_name}} as {{script.Func}} using 'script.py';`
```
Comment (waldyrious) [inline pages/common/odps-func.md]: Sorry I missed this before, but shouldn't `script.py` be wrapped in `{{}}` just like `package.jar` above?

Comment (onesuper) [inline pages/common/odps-func.md]: @waldyrious thanks!

**Comment by waldyrious** [comment]
LGTM 👍 :)

**Comment by felixonmars** [comment]
:ok_hand: Thanks!

**Pull request merged**